app/controllers/admin/items_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::ItemsController < Admin::BaseController
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
```

Comment:
we didn't have a controller that was for admin/merchants/:id/items, so we added a controller. this allows for admin to see a different page, originally the merchant items was redirecting back to the items index

Reply:
i like it